tests/trainer/test_checkpoint.py

```diff
@@ -247,10 +252,18 @@ def test_load_weights(
     (0, 1),
     (1, 0),
 ])
-@pytest.mark.parametrize("model_name", [None, "resnet50_synthetic", "gpt2_52m"])
+@pytest.mark.parametrize(
+    "model_name",
+    [
+        None,
+        # "resnet50_synthetic",
```

Comment:
curious why are these commented out?

Reply:
They were flaky. I checked and they're flaky on dev too. We hadn't noticed because they're only use on GPU testing, which we don't automate.